Lib/test/test_grammar.py

```diff
@@ -1453,6 +1453,8 @@ def test_try(self):
         try: 1/0
         except (EOFError, TypeError, ZeroDivisionError): pass
         try: 1/0
+        except EOFError, TypeError, ZeroDivisionError: pass
```

Comment:
Is it worth adding a test that `except EOFError, TypeError, ZeroDivisionError as msg` raises a SyntaxError?

Reply:
There is a test for that already in `test_syntax`